core/src/main/java/org/elasticsearch/common/geo/builders/ShapeBuilders.java

```diff
@@ -26,7 +26,11 @@
 
 /**
  * A collection of static methods for creating ShapeBuilders.
+ *
+ * @deprecated this class is replaced by each builder's constructor
+ * e.g. instead of using {@code ShapeBuilders.newMultiPoint} use {@code new MultiPointBuilder()}
  */
+@Deprecated
```

Comment:
`public static void register(List<Entry> namedWriteables)` is unused and can probably already be removed, should only be used internally.